_specs/ecip-1000.md

```diff
@@ -353,6 +353,12 @@ In addition, it is recommended that literal code included in the ECIP be dual-li
 * Some ECIPs, especially consensus layer, may include literal code in the ECIP itself which may not be available under the exact license terms of the ECIP.
 * Despite this, not all software licenses would be acceptable for content included in ECIPs.
 
+# Appendix: Irregular Process Transitions
+
+As a decentralized community, it is important that for all governance actions, especially in ECIP, the established process is followed. In this section, we document irregular process transitions, and give special permissions to them, to state that they followed the process. Note that this is essentially similar to theDAO hard fork in Ethereum, and should only be added in extreme cases.
+
+* In [PR#199](https://github.com/ethereumclassic/ECIPs/pull/199), Afri Schoedon merged a PR moving ECIP-1061 to "Last Call" status. At the time, Afri dismissed a review of clear objections from ECIP editor Wei Tang, without waiting for consultations of the community, or resolutions with the review submitter. We here declare that this particular action from Afri followed the ECIP process. 
```

Comment:
Most importantly, @sorpaas, __you need to remove these and all mentions of personal names__ which may or may not actually be associated with, but are intended to be associated with, @soc1c, at his request (see link below). Having done so represents not only poor taste, but also a very plausible case of slander, and cannot be tolerated in any context. 

If we are to establish Irregular Process Transitions, I think we should make the following effort:
- Step back and consider if we actually do, understanding that the core concept is paradoxical, ie. _Did not follow process but followed process._
- Establish beyond reasonable doubt that an event (which must be clearly defined) indeed qualifies as such, with something like a "rough consensus." Hum....?
- Establish (somehow, miraculously) an as-near-to-objective-as-possible and as-near-to-comprehensive-as-possible account the bespoke un-/impossible process.

In the case that these hypothetical procedures and qualifications (as I've roughly outlined) for an Appendix and its contents can be met, then we might have something worth permanently and canonically documenting.

In the case these are not or cannot be met, then we run a very great risk of seeing this content and it's like used unilaterally as an unconstructive tool to allege grudges under ambiguous and/or subjective circumstances.  

As it stands, I see the proposed content sadly as the latter case, where __conditions are insufficient to qualify as an objective representation of affairs__ (which stands to reason, considering it's provenance from an alleged aggrieved party in the case cited).

- https://github.com/ethereumclassic/ECIPs/pull/220#issuecomment-560103503

Reply:
The real name of @soc1c is publicly available in the editors section of ECIP-1000. Should we remove it from that place first, if @soc1c does not want himself to be associated as Afri?

I disagree with the rest of the points you’re trying to make in the post. Irregular transition is there because it’s.. irregular. The thing is, we have to have irregular state transition if we want past editing actions to be justified. Otherwise, specs like 1061 will just become invalid. I guess you do not want to see that.

I do agree that irregular transitions should only be added in really extreme cases, like what we have now. I don’t think we can give objective criteria for it — the same as we cannot give objective criteria for theDAO.